aws_devstack.md

```diff
@@ -0,0 +1,85 @@
+# Installing Romana on AWS EC2 with Devstack
+
+The setup described below has been tested on Ubuntu 14.04 LTS, but should work similarly on other Linux or Mac OS X environments.
+You may need to install additional development tools.
+
+If you do not wish to install additional tools in your host, you can create a VM and execute the installation steps from there.
+
+# Prepare
+
+To run this installation, you will need
+* an AWS account. You can create one at [http://aws.amazon.com](http://aws.amazon.com)
+* credentials for an [AWS IAM](https://console.aws.amazon.com/iam/home) User/Role that permits creating EC2 instances
+* [ansible](https://www.ansible.com) and supporting python tools / libraries
+
+## Set up Ansible
+
+```bash
+# Ubuntu
+sudo apt-get install git python-pip python-dev
+sudo pip install ansible==1.9.4 boto awscli netaddr
+
+# OS X
+sudo easy_install pip
+sudo pip install ansible==1.9.4 boto awscli netaddr
+```
+
+## Set up AWS tools
+
+```bash
+aws configure
+```
+
+This will prompt you for your AWS Access Key ID, AWS Secret Key ID and Region.
```

Comment:
We could "document" it by opening an issue, see if users add 'me too' comments to it.